climada/entity/exposures/litpop/litpop.py

```diff
@@ -125,6 +129,10 @@ def set_countries(self, countries, res_arcsec=30, exponents=(1,1),
         Raises
         ------
         ValueError
+
+        Returns
+        -------
+        LitPop
```

Comment:
Make sure to mention in the docstring that the constructor method returns a (new) object. Go through the existing docstrings and amend any references to the deprecated approach, e.g. "clearing" or "filling" of the object.